app/script/util/Emoji.js

```diff
@@ -22,15 +22,31 @@
+  .replace(/U\+/g, '')
+  .split(', ')
+  .reduce((list, codepoint) => {
+    if (codepoint.indexOf('-') === -1) {
+      list.push(String.fromCodePoint(`0x${codepoint}`));
+    } else {
+      const [start, end] = codepoint.split('-').map((code) => parseInt(code, 16));
+      for (let code = start; code <= end; code++) {
+        list.push(String.fromCodePoint(`0x${code.toString(16)}`));
```

Comment:
Let's start applying the [no-magic-numbers](http://eslint.org/docs/rules/no-magic-numbers) to the new code :)